tools/tiller-hack/tiller-hack.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
```

Comment:
please don't introduce another logging framework into istio - i think vanilla glog is fine here and istio uses uber zap.

Reply:
well, glog does not have Infof, Warnf etc, but to be consistent with the rest of istio will switch to glog.